ProductApp/src/main/java/com/stack/jspdemo/controller/ProductController.java

```diff
@@ -0,0 +1,39 @@
+package com.stack.jspdemo.controller;
+
+import java.net.URI;
+import java.util.List;
+
+import javax.validation.Valid;
+
+import com.stack.jspdemo.model.Product;
+import com.stack.jspdemo.service.ProductService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+
+@RestController
+public class ProductController {
+
+	@Autowired
```

Comment:
Same here about using `@Autowired` annotation

Reply:
Delete this file because it was not necessary anymore